src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx

```diff
@@ -140,11 +142,33 @@ export default class InstalledIntegrationInDirectory extends React.Component<Pro
               <IntegrationItem integration={integration} />
             </IntegrationItemBox>
             <div>
-              {
+              {showSlackAlert && (
                 <Tooltip
-                  disabled={this.hasConfiguration()}
-                  position="left"
-                  title="Integration not configurable"
+                  disabled={hasAccess}
+                  title={t(
+                    'You must be an organization owner, manager or admin to re-authenticate'
+                  )}
+                >
```

Comment:
Button has a `title` attribute which you can use for the tootlip actually

Reply:
@EvanPurkhiser The problem with that approach is `title` always rendered. And type for `title` is `string`. Meaning I can't do something like this:
```
title={!hasAccess && t( 'You must be an organization owner, manager or admin to re-authenticate')}`
```
I could use an empty string but that seems a bit hacky to me

Reply:
We should fix the typing on it in that case

Reply:
But this is fine for now.